src/pd/mod.rs

```diff
@@ -146,4 +146,13 @@ pub trait PdClient: Send + Sync {
     fn report_split(&self, left: metapb::Region, right: metapb::Region) -> PdFuture<()>;
 }
 
+// PdClient and some other manage commands.
+pub trait PdCtlClient: PdClient {
+    // Get region and its leader by id.
+    fn get_region_and_leader_by_id(
```

Comment:
I prefer adding a `get_leader` function and we use these separately.

Reply:
If we separate them, we must call `GetRegion` API twice, which maybe cause two responses have different content. Maybe it's a problem?